multirun/multirun.go

```diff
@@ -101,11 +103,11 @@ func (m multirun) boundedExecution(ctx context.Context) error {
 	return nil
 }
 
-func (m multirun) spawnWorker(ctx context.Context, commands <-chan command) error {
+func (m multirun) spawnWorker(ctx context.Context, commands <-chan command, errs chan<- error) bool {
 	for cmd := range commands {
 		select {
 		case <-ctx.Done():
-			return ctx.Err()
+			return false
```

Comment:
Shouldn't it push the error into the `errs` here and return `true`?

Reply:
Cool, I updated the code to do this.